src/mbgl/programs/programs.hpp

```diff
@@ -13,21 +13,23 @@ namespace mbgl {
 
 class Programs {
 public:
-    Programs(gl::Context& context, const ProgramParameters& programParameters)
-        : circle(context, programParameters),
-          fill(context, programParameters),
-          fillPattern(context, programParameters),
-          fillOutline(context, programParameters),
-          fillOutlinePattern(context, programParameters),
-          line(context, programParameters),
-          lineSDF(context, programParameters),
-          linePattern(context, programParameters),
-          raster(context, programParameters),
-          symbolIcon(context, programParameters),
-          symbolIconSDF(context, programParameters),
-          symbolGlyph(context, programParameters),
-          debug(context, ProgramParameters(programParameters.pixelRatio, false)),
-          collisionBox(context, ProgramParameters(programParameters.pixelRatio, false)) {
+    Programs(gl::Context& context,
+             const ProgramParameters& programParameters,
```

Comment:
yeah; but for the last two shaders we are constructing new ProgramParameter objects as well...